src/lib/server/xlat_func.c

```diff
@@ -199,12 +199,34 @@ int xlat_fmt_to_cursor(TALLOC_CTX *ctx, fr_cursor_t **out,
 /** Print length of its RHS.
  *
  */
-static ssize_t xlat_strlen(UNUSED TALLOC_CTX *ctx, char **out, size_t outlen,
-			   UNUSED void const *mod_inst, UNUSED void const *xlat_inst,
-			   UNUSED REQUEST *request, char const *fmt)
+static xlat_action_t xlat_strlen(TALLOC_CTX *ctx, fr_cursor_t *out,
+				 UNUSED REQUEST *request, UNUSED void const *xlat_inst,
```

Comment:
Request should be marked as UNUSED if you're using debugging macros, because they get logging info from the request

Reply:
I'll merge this one and fix it on my side